test-app/app/components/docs/components/phone-input/all-options.hbs

```diff
@@ -0,0 +1,129 @@
+<h2>Basic Options</h2>
+<DocsDemo as |demo|>
+  <demo.example @name='phone-input-all-options.hbs'>
+
+    <p><PhoneInput
+        @initialCountry='us'
+        @number={{this.number}}
+        @update={{this.handleUpdate}}
+      /></p>
+
+    <p>The internal phone number you'll want to persist on your backend is
+      <em>\{{this.number}}: </em><strong>{{this.number}}</strong>.</p>
+
+    <p>In some cases, you may want to persist some additional metadata. Are
+      available:</p>
```

Comment:
I find "Are available" not very grammatical. Maybe you can rephrase it to something like: "Following properties are available:"?

Reply:
We are going to replace docs with markdown in the next PR and also I'd really like to avoid mixing docs changes with the typescript migration that is done in this PR if that's ok. WDYT?

Reply:
Absolutely OK, yes!